.github/workflows/ci-test.yaml

```diff
@@ -106,7 +106,6 @@ jobs:
           # TODO: Load only what is needed
       - name: Load images and verify
         run: |
-          docker load -i build/testimage.tar
```

Comment:
Added run-id to all the test image names and just forgot this one, hence the error in the last push from master.